### PR TITLE
implemented bookDto, bookController/ Added bookMapper

### PR DESCRIPTION
**Pull request by SaiyaJ1N**

### CODE REVIEW

File: src/main/java/com/onlinebookshop/controller/BookController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/books")
```
Comment (OlegStashkiv) [inline src/main/java/com/onlinebookshop/controller/BookController.java:17]: By condition it should be:
Book Endpoints: /api/books

**Review by OlegStashkiv — APPROVED** [review]
Great solution. Approve. But I still recommend correcting the comments.

**Review by ohatsenko — APPROVED** [review]

File: src/main/java/com/onlinebookshop/controller/BookController.java (hunk 1 above)
Comment (vovasalyha) [reply src/main/java/com/onlinebookshop/controller/BookController.java:17]: Agree with Oleg

**Review by vovasalyha — APPROVED** [review]
GJ! Please add /api prefix.

**Pull request merged**